### PR TITLE
etcd, system container: Fix unable to mask service etcd_container

### PR DESCRIPTION
**Pull request by giuseppe**
Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1462087

### CODE REVIEW

**Comment by giuseppe** [comment]
aos-ci-test

File: roles/etcd/tasks/system_container.yml [hunk 1]
```diff
@@ -24,23 +24,30 @@
...
   register: task_result
   failed_when: task_result|failed and 'could not' not in task_result.msg|lower
 
+- name: Remove etcd_container.service
+  file:
+    path: /etc/systemd/system/etcd_container.service
+    state: absent
+
+- name: Systemd reload configuration
+  systemd: name=etcd_container daemon_reload=yes
```
Comment (sdodson) [inline roles/etcd/tasks/system_container.yml]: I don't want to nak this becase you'd have to go back through the ci jobs, but lets make sure that we use the same format that in the future we use the multi-line format.
```
- name: Systemd reload configuration
  systemd:
    name: etcd_container
    daemon_reload: yes
```

**Comment by sdodson** [comment]
[merge][severity: bug]

**Comment by sdodson** [comment]
[merge][severity:blocker]

**Pull request merged**